### PR TITLE
test: add tests for HIP 755 and enable feature flag

### LINKED ISSUE #16889: Write bdd tests for coverage

### PR DESCRIPTION
**Pull request by lukelee-sl**
**Description**:
Add unit and BDD tests for HIP 755

**Related issue(s)**:

Fixes #16889 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by stoyanov-st — DISMISSED** [review]
LG ty @lukelee-sl

**Review by Nana-EC — APPROVED** [review]

**Review by stoyanov-st — DISMISSED** [review]

File: hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java [hunk 1]
```diff
@@ -83,7 +83,7 @@ public record ContractsConfig(
         @ConfigProperty(value = "systemContract.scheduleService.signSchedule.enabled", defaultValue = "true")
                 @NetworkProperty
                 boolean systemContractSignScheduleEnabled,
-        @ConfigProperty(value = "systemContract.scheduleService.authorizeSchedule.enabled", defaultValue = "false")
+        @ConfigProperty(value = "systemContract.scheduleService.authorizeSchedule.enabled", defaultValue = "true")
```
Comment (david-bakin-sl) [inline hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java:86]: This change is important and should be called out in the PR description.  Might also mean that the title "test" is not right?

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM from `hedera-services` file changes. Thanks @lukelee-sl

File: hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java (hunk 1 above)
Comment (Neeharika-Sompalli) [inline hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java:86]: The issue says just increase test coverage. But we are enabling this feature right ?

**Review by Nana-EC — APPROVED** [review]

**Review by david-bakin-sl — APPROVED** [review]
LGTM

File: hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java (hunk 1 above)
Comment (lukelee-sl) [reply hedera-node/hedera-config/src/main/java/com/hedera/node/config/data/ContractsConfig.java:86]: Agreed.  updated description and title.

**Pull request merged**